File: Data Structures & Algorithms/implement-stack-using-queues/submission-0.py
class MyStack:
    st=[]
    def __init__(self):
        self.st=[]

    def push(self, x: int) -> None:
        self.st.insert(0,x)

    def pop(self) -> int:
        # ele=self.st[0]
        # del self.st[0]
        return self.st.pop(0)

    def top(self) -> int:
        return self.st[0]

    def empty(self) -> bool:
        if self.st:
            return False
        else:
            return True


# Your MyStack object will be instantiated and called as such:
# obj = MyStack()
# obj.push(x)
# param_2 = obj.pop()
# param_3 = obj.top()
# param_4 = obj.empty()

File: Data Structures & Algorithms/design-hashmap/submission-0.cpp
class MyHashMap {
    vector<int> _hm;
public:
    MyHashMap():_hm(1000001,-1) {

    }
    
    void put(int key, int value) {
        _hm[key]=value;
    }
    
    int get(int key) {
        return _hm[key];
    }
    
    void remove(int key) {
       _hm[key]=-1;
    }
};

/**
 * Your MyHashMap object will be instantiated and called as such:
 * MyHashMap* obj = new MyHashMap();
 * obj->put(key,value);
 * int param_2 = obj->get(key);
 * obj->remove(key);
 */